dockerfiles/init/modules/openshift/files/scripts/multi-user/wait_until_postgres_is_available.sh

```diff
@@ -16,10 +16,10 @@ command -v jq >/dev/null 2>&1 || { echo >&2 "[CHE] [ERROR] Command line tool jq
 available=$(oc get dc postgres -o json | jq '.status.conditions[] | select(.type == "Available") | .status')
 progressing=$(oc get dc postgres -o json | jq '.status.conditions[] | select(.type == "Progressing") | .status')
 
-DEPLOYMENT_TIMEOUT_SEC=120
+DEPLOYMENT_TIMEOUT_SEC=1200
 POLLING_INTERVAL_SEC=5
 end=$((SECONDS+DEPLOYMENT_TIMEOUT_SEC))
-while [ "${available}" != "\"True\"" ] && [ ${SECONDS} -lt ${end} ]; do
+while [[ "${available}" != "\"True\"" || "${progressing}" != "\"True\"" ]] && [ ${SECONDS} -lt ${end} ]; do
```

Comment:
The second part that you're adding: `|| "${progressing}" != "\"True\"" ` was removed some commits ago, because, in some use cases, the `progressing` status never comes to `True` even when the `available` status becomes `True`.  

Could you precise what is the issue this addition is trying to fix ?

Reply:
@riuvshin helped me here and he said that when Postgres is available but not progressing is not enough to consider Postgres is fully online.

Reply:
I'm really surprised about that. @riuvshin could you give more details about that ? If I remember correctly, we had encountered cases when `processing` *never* comes to `True`, even when the Postgres server is correctly up and running.

Reply:
@davidfestal hm, no... As I remember our issue was that progressing was in 3 states:
 **empty**, **unknown**, **true**;
 while script was expected only 2 states **emtpy** and **true** and it was exiting when it was not **empty** _but_ also it wasn't yet **true**

Reply:
OK, then